etna/datasets/tsdataset.py

```diff
@@ -307,20 +244,20 @@ def make_future(self, future_steps: int, tail_steps: int = 0) -> "TSDataset":
                             f"NaN-s will be used for missing values"
                         )
 
-        if self.transforms is not None:
-            for transform in self.transforms:
-                tslogger.log(f"Transform {repr(transform)} is applied to dataset")
-                df = transform.transform(df)
+        ts = TSDataset(df=df, freq=self.freq)
```

Comment:
Is it working? What about regressors and known future? It seems like we are losing this information here.

Reply:
Added comment